assets/styles/components/_integrations.scss

```diff
@@ -35,6 +35,19 @@ $screen-md-max: ($screen-lg-min - 1) !default;
     height: auto;
   }
 
+  table {
+    tr {
+        th:empty {
+            display: none;
+        }
+
+        td {
+            word-break: break-word;
+            width: 50%;
+        }
+    }
+  }
+
```

Comment:
need the CSS changes here as the metrics tables used to be driven by a shortcode but are now pure markdown (so cant drop in custom classes/no html)